### ISSUE TITLE
Bootstrap Close icon helper class is styled incorrectly on hover when used with uiv Button

### ISSUE THREAD

**Issue by arrancardnell** [wxsms/uiv#307]
**Describe the bug**

The built-in Close icon helper class for Bootstrap shows a button hover effect when used with a uiv Button.

**To Reproduce**
1. Add Bootstrap Close icon helper class:

```HTML
<button type="button" class="close" aria-label="Close"><span aria-hidden="true">&times;</span></button>
```

2. Edit button to use uiv button:

```HTML
<Btn class="close" aria-label="Close"><span aria-hidden="true">&times;</span></Btn>
```

**Expected behavior**

Cross should appear darker on hover with no change to the background behind the button.

![image](https://user-images.githubusercontent.com/6402515/69166643-7a14d780-0aeb-11ea-8758-a8b388de29b3.png)

**Actual behavior**

Cross is darker but background behind button changes colour.

![image](https://user-images.githubusercontent.com/6402515/69166807-c102cd00-0aeb-11ea-88b8-4c77a7f6a2c7.png)

**Desktop (please complete the following information):**
 - OS: Windows 7
 - Browser: Firefox / Chrome
 - Version: 70.0.1 (64-bit) / 78.0.3904.108 (Official Build) (64-bit)

**Comment by wxsms** [comment]
TLDR: btn component is not designed for this usage.

It's because btn component added `btn btn-default` class by default. When you pass a `class="close"` it becomes a merged result, not replacement. In this case the proper approach should be adding a new `close` type to it. However, since this button type is not as normal as others, and you can achive it with one single class, I think it's easy enough to use with the native button tag either. :smile:
